packages/kbn-apm-config-loader/src/config.ts

```diff
@@ -156,23 +160,6 @@ export class ApmConfiguration {
     return this.rawKibanaConfig?.elastic?.apm ?? {};
   }
 
-  /**
-   * Get the configuration from the apm.dev.js file, supersedes config
-   * from the --config file, disabled when running the distributable
-   */
-  private getDevConfig(): AgentConfigOptions {
```

Comment:
Using `kibana.dev.yml` is a more standard and intuitive way.

Reply:
We knew we'd be changing where we read the config from [eventually](https://github.com/elastic/kibana/pull/77855#discussion_r492373174), just didn't have support for config in `kibana.dev.yml` at the time the package was added.